js/interface.js

```diff
@@ -369,7 +369,12 @@ function fetchCurrentDataSourceEntries(entries) {
         return Promise.resolve(entries);
       }
 
-      return source.find({}).catch(function() {
+      return source.find({
+        orderBy: [
```

Comment:
As per tech spec (https://weboo.atlassian.net/browse/XOR-563?focusedCommentId=36803), the column should be called `order`, not `orderBy`.

Reply:
In the existing endpoint `order` name is already taken so added `orderBy` [here](https://github.com/Fliplet/fliplet-api/blob/%40Smoholxor/FIX-XOR-563/routes/v1/data-sources.js#:~:text=%5B%27id%27%2C%20%27createdAt%27%2C%20%27updatedAt%27%2C%20%27order%27%5D.forEach((key)%20%3D%3E%20%7B)

Reply:
yes we supposed to add the same however body is already having `order `field e.g here(https://github.com/Fliplet/fliplet-api/blob/%40Smoholxor/FIX-XOR-563/routes/v1/data-sources.js#:~:text=%5B%27id%27%2C%20%27createdAt%27%2C%20%27updatedAt%27%2C%20%27order%27%5D.forEach((key)%20%3D%3E%20%7B) so to avoid conflict with name added `orderBy` instead of `order`

Reply:
@squallstar as per https://github.com/Fliplet/fliplet-api/pull/5528#discussion_r990591664 comment  I do changes accordingly please review